src/pages/Admin/Charity/EditProfile/schema.ts

```diff
@@ -16,6 +16,7 @@ export const VALID_MIME_TYPES: ImageMIMEType[] = [
 ];
 
 export const MAX_SIZE_IN_BYTES = 1e6;
+export const MAX_CHARS = 4000;
```

Comment:
bug: chars is more than `4000` but can still submit

Reply:
Can you post repro steps?

Reply:
> Can you post repro steps?

1. go to edit-profile
2. input more than 4000 chars on overview
3. submit

Reply:
This also seems to be a bug inherited from `master`, so outside the scope of this PR.

Reply:
Created a ticket https://linear.app/better-giving/issue/BG-1173/edit-profile-stop-submission-when-overview-contains-more-than-the

Reply:
you applied `richTextContent` to `EditProfile`, no ? So we just accept that it doesn't work (can't detect error) and approve this PR ? Are we now okay to ship something that doesn't work? @SovereignAndrey  - with the reason the we could just create another PR to fix it. Isn't the point of this PR is to fix validation concern? 

Even new programs that I created exceeds `500` chars , and it submitted (not catched by `richTextContent`).

Reply:
created pr to do just this 
* https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2751

Reply:
@ap-justin there's no need to make a big deal out of this. I fixed the issue of RichTextEditor not allowing the max. number of characters it's expected to allow and not to fix an unrelated issue where a form would submit even when one of its fields is invalid. Also see https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2744#discussion_r1505455417